net/mbusd/files/mbusd.init

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2019 OpenWrt.org
```

Comment:
this was also an invalid copyright assignment

Reply:
Removed